web/src/components/CityContentFooter.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { DISCLAIMER_ROUTE, LICENSES_ROUTE, pathnameFromRouteInformation } from 'api-client'
+
+import buildConfig from '../constants/buildConfig'
+import CleanLink from './CleanLink'
+import Footer from './Footer'
+
+const SidebarFooterContainer = styled.div`
+  width: 100%;
+  margin: -10px 12px 0 23px;
```

Comment:
this looks like magic :)
i think instead of that margin i would use a padding. Even the spacings don't look equal

![image](https://user-images.githubusercontent.com/37902063/195548879-0c22732d-607d-41d2-9f80-e4314e5d7b42.png)
```suggestion
  padding: 0 24px; 
```